apluslms-yamlidator/apluslms_yamlidator/document.py

```diff
@@ -2,13 +2,17 @@
 from abc import ABCMeta
 from copy import deepcopy
 from hashlib import sha1
+from itertools import chain, zip_longest
```

Comment:
`zip_longest` is imported, but not used.  Was is supposed to be?

Reply:
it was at one point :') put it back